### ISSUE TITLE
Consider making this project part of official GNOME application sets

### ISSUE THREAD

**Issue by alive4ever** [gnome-mpv/gnome-mpv#226]
Since gnome-mpv isn't included in official GNOME application, it doesn't come with gnome-extras package in most distributions.

I think this project is eligible for inclusion in GNOME, so that the project reach more audience.

A good starting point is [GNOME contribution wiki](https://wiki.gnome.org/Community/Contribute) and [GNOME get involved guide](https://www.gnome.org/get-involved).

**Comment by TingPing** [comment]
Most distros would never ships gnome-mpv by default. They have a huge investment in Totem and GStreamer and the whole patent situation of ffmpeg often makes it less than usable.

**Comment by gnome-mpv** [comment]
I don't think GNOME would be interested in including gnome-mpv due to feature overlap with Totem and patent issues with ffmpeg as TingPing said, but I'll ask GNOME people later when I have the chance. I'm a little busy right now.

**Comment by infinity0** [comment]
GNOME have very pretentious design guidelines that one must adhere to, that makes applications much less usable. TBH I would be happier if this were kept out of the GNOME official application set and the gnome-mpv maintainers could exercise their own independent judgement in how to design their application.

**Comment by TingPing** [comment]
> GNOME have very pretentious design guidelines that one must adhere to

As the maintainer you can basically do whatever you want really they just won't consider your app to be part of the core moduleset that distros should ship installed by default if you fail to live up to the experience they want. With that said anyway GnomeMpv is fairly simple and checks most of the boxes for Gnome designs. As I mentioned though Totem is part of their core apps and no other video player would replace that.

**Comment by infinity0** [comment]
"won't consider if" means the same thing as "must adhere to". I'm not sure what you thought I meant instead. As a user I actively *don't* want applications to "live up to" the experience they want. My wants are very very different.

**Comment by TingPing** [comment]
I'm saying the worst they would do is say "Distros you shouldn't install this by default" which they will never do anyway. They don't suddenly take over your project and change its direction.

Anyway the main point against this is not Gnome but rather what does *this* project gain. The project hosting is irrelevant as Github seems fine, translation infrastructure would be nice, and it could possibly become more likely for distros to pick it up. Not major wins IMO.

**Comment by infinity0** [comment]
They don't suddenly take over your project, for sure - rather, what I meant was: if gnome-mpv makes "being a core GNOME application" as #1 priority, then this is going to affect how the project is run and how the application is developed, even if GNOME are not directly controlling it. I wouldn't like this to happen either.

**Comment by TingPing** [comment]
Sure, I don't think anybody involved wants that.

**Comment by jannuary** [comment]
Huh, it's been 5 years


Anyhow, [GNOME Circle](https://circle.gnome.org/) is a thing now

**Comment by emanuelserpa** [comment]
Yeah, I think Celluloid deserves to be in GNOME Circle. Pretty good job @gnome-mpv is doing here.

**Comment by TingPing** [comment]
Another thing that has changed in 5 years is Totem has been kept back on GTK3 because of technical debt, so there is room for a new video player.

**Comment by Scotty-Trees** [comment]
Any thoughts in 2024 to consider Gnome Circle? Totem is dated and I haven't used it in year, plus it's still using GTK3.

**Comment by gnome-mpv** [comment]
> Any thoughts in 2024 to consider Gnome Circle? Totem is dated and I haven't used it in years*, plus it's still using GTK3.

I'm considering applying for GNOME Circle at some point. Looking through the [requirements](https://gitlab.gnome.org/Teams/Releng/AppOrganization/-/blob/main/AppCriteria.md), we still need to set up CI and include a .doap file before we're ready to apply.

**Comment by TingPing** [comment]
I don't mind setting up CI, but you'll need to do that on GitLab and not here.

**Comment by gnome-mpv** [comment]
> I don't mind setting up CI, but you'll need to do that on GitLab and not here.

I can set one up here later. Moving the repo to GitLab is going to be pretty disruptive, so I don't want to do that yet.
